### PR TITLE
Completed es6-template-literals exercise - Joe Dei Rossi

### PR DESCRIPTION
**Pull request by JoeDeiRossi**
image

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log(`LearningFuze: ${address}`);
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Do not change anything which is not concatenation:

---

image

---

image

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = fruit + item;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Missed a concatenation.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: es6-template-literals/index.html (hunk 1 above)
Comment (JoeDeiRossi) [reply es6-template-literals/index.html]: Is there a separate term for this? I thought it was just a different way of concatenating.

File: es6-template-literals/index.html (hunk 2 above)
Comment (JoeDeiRossi) [reply es6-template-literals/index.html]: Practically, is there a reason to use template literals for something like this where there are only variables being concatenated, not strings?

File: es6-template-literals/index.html (hunk 1 above)
Comment (TimelikeClosure) [reply es6-template-literals/index.html]: @JoeDeiRossi Passing the `console.log` method two arguments is different from a single concatenated argument.

File: es6-template-literals/index.html (hunk 2 above)
Comment (TimelikeClosure) [reply es6-template-literals/index.html]: Practically, the code may be more or less readable, depending on what it's for. However, this exercise asks you to convert _all_ concatenations to template literals for the practice, so please do so.

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**